packages/schema-views/src/views/types/function.tsx

```diff
@@ -8,34 +8,25 @@ import style from './type.st.css';
 
 export const FunctionTypeView: React.FunctionComponent<ISchemaViewProps> = (props) => {
   const {schema} = props;
-  const restArg = schema.restArgument;
   const required = schema.requiredArguments || [];
-  const returnTypeJsx = isVoid(schema.returns) ?
-    'void' :
-    (
-      <BaseView
-        schemaRegistry={props.schemaRegistry}
-        viewRegistry={props.viewRegistry}
-        schema={schema.returns}
-      />
-    );
-
-  const args = schema.arguments.map((arg: Schema) => {
+  const args = schema.arguments.map((arg: Schema, index: number) => {
     const optional = required.includes(arg.name) ? '' : '?';
-    const defaultValue = arg.default;
+    const argName = arg.name + optional;
     return [
-      arg.name + optional + ': ',
+      argName + ': ',
       (
         <BaseView
           schemaRegistry={props.schemaRegistry}
           viewRegistry={props.viewRegistry}
           schema={arg}
         />
       ),
-      defaultValue ? ` = ${defaultValue}` : ''
+      arg.hasOwnProperty('default') ? ` = ${JSON.stringify(arg.default)}` : (
```

Comment:
I think `'default' in args` would express the intent more clearly. I was confused for a moment thinking about why would we want to exclude properties coming from the prototype, to only realize that the actual intent is to correctly handle default when it's falsy, something that I missed when I initially wrote this function.